openedx_webhooks/info.py

```diff
@@ -48,13 +48,18 @@ def get_orgs(key):
     orgs = get_orgs_file()
     return set(o for o, info in orgs.items() if info.get(key, False))
 
-def get_person_time_of_pull_request(person, created_at):
+def get_updated_person(person, updated_time):
```

Comment:
I'm not sure how finely detailed to get here.  Ideally the name of the function would include the idea of "date in the past" or something. "Updated" is pretty vague, though the docstring covers it, so....